source/api_documentation/call_api/_get_call_transcript_caller_agent_block.rst

```diff
@@ -0,0 +1,24 @@
+.. container:: endpoint-long-description
+
+  .. rubric:: Examples
+
+  Get transcript for the given call record `CUID-33` with both agent and caller included in a single block for each.
+
+  Endpoint:
+
+  ``https://mynetwork.invoca.net/call/transcript/CUID-33?transcript_format=agent_caller_block``
+
+  Format: application/json
+
+  Response Code: 200
+
+  Response Body:
+
+  .. code-block:: json
+
+     [
+       {
```

Comment:
So in cases like this, we will actually get the caller and agent in different objects. Sorry I didn't include examples yet. So it would look like this instead: 

```
[
   { agent: "Which Color?, Okay I will look into that, Goodbye!" }
   { caller: "Orange, Awesome thank you, Have a good one"  }
]
```

Reply:
This is the behavior that was confirmed with Nash, but I could double check with him on this one.

Reply:
Confirmed we will keep the existing format providing caller and agent in their own blocks